src/ui-kit/types.ts

```diff
@@ -344,3 +344,32 @@ export interface UploadedFileData {
      */
   icon: any;
 }
+
+export interface UploadFileActionModalConfig {
+    /**
+     * sets the upload modal title
+     */
+    title?: string,
+    /**
+     * sets the upload modal description
```

Comment:
the description is no longer being used, can we remove it in the meantime?

Reply:
I was using in ui-kit-demo.page